apps/api-extractor/src/generators/ApiReportGenerator.ts

```diff
@@ -427,6 +431,12 @@ export class ApiReportGenerator {
     }
   }
 
+  private static _shouldIncludeInReport(astDeclaration: AstDeclaration): boolean {
+    // Private declarations are not included in the API report
```

Comment:
Move to a TSDoc comment for the function?

Reply:
The comment describes the current implementation, not the behavior of the function.